### PR TITLE
Cache rebased

### PR DESCRIPTION
**Pull request by harikt**
For https://github.com/cakephp/migrations/pull/309

### CODE REVIEW

File: src/Command/CacheBuild.php [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+class CacheBuild extends Command
+{
+    use SchemaTrait;
+
+    protected function configure()
+    {
+        $this
+            ->setName('orm-cache-build')
+            ->setDescription('Build all metadata caches for the connection. If a table name is provided, only that table will be cached.')
```
Comment (harikt) [inline src/Command/CacheBuild.php]: I wished if the @stickler-ci could fix it than complaining.

**Comment by harikt** [comment]
Is there anything I can do to make this merge ?

File: src/Command/CacheBuild.php (hunk 1 above)
Comment (lorenzo) [inline src/Command/CacheBuild.php]: @markstory not a bad idea ^

**Comment by lorenzo** [comment]
I'll merge now but not doing a release until I confirm it does not create a big conflict with the shells provided by the cake core

**Comment by lorenzo** [comment]
@harikt thanks so much!!

**Comment by harikt** [comment]
@lorenzo thank you for merging this.

> I confirm it does not create a big conflict with the shells provided by the cake core

I don't believe so. But you can test things before the release :-) .

> thanks so much!!

You are very welcome. And thank you once again for the merge. I hate to see PR's waiting for days :-) .

**Pull request merged**